### ISSUE TITLE
Lack of outlines/contrast in GUI components

### ISSUE THREAD

**Issue by Steve3854** [ChrisRM/material-theme-jetbrains#1774]
### Summary

Many of the themes have a noticeable lack of outlines and color contrast in the GUI. Definitely not a major issue, but looks weird and detracts from the plugin.

### Expected behavior
Darcula (example of a default theme with outlines/contrast)
![Darcula](https://i.ibb.co/wdPBBBy/06-06-21-21-22-30-847-pycharm64.png)

### Actual behavior
Atom One Dark Material (example of a Material theme)
![Atom One Dark](https://i.ibb.co/QCcYr53/06-06-21-21-22-51-005-pycharm64-edited.png)

### Steps to reproduce
1. Install Material Theme
2.  Go to "Tools", "Material Theme", "Material Theme Chooser", and "Atom One Dark Theme" (though this problem exists in other themes too)

### Environment

- PyCharm 2021.1.2 (Professional Edition)
- Build #PY-211.7442.45, built on June 1, 2021
- Licensed to Stephen Liu
- Subscription is active until June 1, 2022.
- For educational use only.
- Runtime version: 11.0.11+9-b1341.57 amd64
- VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
- Windows 10 10.0
- GC: G1 Young Generation, G1 Old Generation
- Memory: 2028M
- Cores: 12
- Registry: ide.new.stripes.ui=true, ide.balloon.shadow.size=0
- Non-Bundled Plugins: com.jetbrains.CyanTheme (1.2), com.mallowigi (36.0), com.markskelton.one-dark-theme (5.1.6), com.chrisrm.idea.MaterialThemeUI (6.6.0)

**Comment by mallowigi** [comment]
Hi,

The lack of outlines/borders is definitely the core principle of the plugin, to provide an interface that looks smooth, without edges and reflections. If you look at the Material Theme in Sublime Text, Atom or Visual Studio you'll see the same principle applied.

Of course, it can't please everyone, that's why the "Contrast Mode" is here to provide a more contrasted alternative. It's a bit different that the VSCode counterpart as VSCode contrasts the title bar, tool window bars and status bars. However in JetBrains' UI these elements are currently using the same color, namely "Panel.background" - but I'm trying to figure out a way to be able to paint them independently.

Now regarding your issue, you are able to change the console and editor background color from the color scheme, so that won't require a drastic change in the themes. And if you want more, you could also use the Custom Theme feature to tweak it even more as you want it (except for the aforementioned issue).

Also, in the plugin page I thought you were referring to the floating tooltip that blends in the background. This is something that I want to change though, because right now it doesn't stand out very much.

**Comment by Steve3854** [comment]
Thanks! Sorry for the additional question but how do I change the console and editor background colors? I know it's in color scheme but I don't know what it's called. Also, I'm a student and am interested in applying for a free license, how do I do that?

**Comment by mallowigi** [comment]
Please send me an email with a screenshot of your JetBrains account showing that you are part of the students program.

For the color scheme, simply look for Settings > Editor > Color scheme > General/Console
